lte/gateway/python/magma/mobilityd/tests/ip_alloc_dhcp_test.py

```diff
@@ -80,7 +80,7 @@ def tearDown(self):
     @unittest.skipIf(os.getuid(), reason="needs root user")
     def test_ip_alloc(self):
         sid1 = "IMSI02917"
-        ip1 = self._dhcp_allocator.alloc_ip_address(sid1)
+        ip1, ignore = self._dhcp_allocator.alloc_ip_address(sid1)
```

Comment:
`ip1, _ = self._dhcp_allocator.alloc_ip_address(sid1)` and same for the following